src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -140,16 +142,28 @@ struct OperationViewModel {
             responses.append(ResponseViewModel(from: response, with: operation))
         }
 
+        var defaultException: ExceptionResponseViewModel?
         for exception in operation.exceptions ?? [] {
-            exceptions.append(ExceptionResponseViewModel(from: exception))
+            let vm = ExceptionResponseViewModel(from: exception)
+            if vm.isDefaultStatusCode {
+                assert(
+                    defaultException == nil,
+                    "Multiple default exception per operation is currently not supported... \(operation.name)"
+                )
+                defaultException = vm
+            } else {
+                exceptions.append(vm)
+            }
         }
 
         var statusCodes = [String]()
         for response in responses {
             statusCodes.append(contentsOf: response.statusCodes)
         }
         for exception in exceptions {
-            statusCodes.append(contentsOf: exception.statusCodes)
+            for statusCode in exception.statusCodes where statusCode != "default" {
```

Comment:
You already filtered out default in the initializer, no?

Reply:
Good catch. Removed this change.